saleor/graphql/order/utils.py

```diff
@@ -97,10 +98,15 @@ def validate_order_lines(order, country):
                 raise ValidationError({"lines": errors})
 
 
+def validate_variants_is_available(order):
+    variants_ids = {line.variant_id for line in order.lines.all()}
+    validate_variants_available_in_channel(
+        variants_ids, order.channel_id, OrderErrorCode
```

Comment:
Wouldn't be more universal if you would pass exact error code, instead of the whole class? Now I will not able use it for different part of API until my error class code will not have an expected error_code. 
IMO this would be more universal:
```
validate_variants_available_in_channel(
variants_ids, order.channel_id, OrderErrorCode.PRODUCT_UNAVAILABLE_FOR_PURCHASE
)
```